iocore/net/QUICNetProcessor_quiche.cc

```diff
@@ -90,8 +90,8 @@ QUICNetProcessor::start(int, size_t stacksize)
   }
 
   quiche_config_set_max_idle_timeout(this->_quiche_config, params->no_activity_timeout_in());
-  quiche_config_set_max_recv_udp_payload_size(this->_quiche_config, 16384);
-  quiche_config_set_max_send_udp_payload_size(this->_quiche_config, 16384);
+  quiche_config_set_max_recv_udp_payload_size(this->_quiche_config, params->get_max_recv_udp_payload_size_in());
+  quiche_config_set_max_send_udp_payload_size(this->_quiche_config, params->get_max_send_udp_payload_size_in());
```

Comment:
in or out?

Reply:
Seems a bit odd that the default value for these config variables is (about) double the previous hard-coded value.

Reply:
It seems to be the default for [this](https://www.rfc-editor.org/rfc/rfc9000.html#section-18.2-4.10.1) transport parameter.
> The default for this parameter is the maximum permitted UDP payload of 65527. Values below 1200 are invalid.well, being hardcoded before means nothing different

Reply:
The hard-coded value means nothing. It could be 12000 or 32768 if I made the commit on a different day.

Reply:
And `_in` is correct for the both here. It's for inbound connections. The `_out` will be used when we support H*3* to  origin.

Reply:
From the names of the the Quiche API functions, it doesn't look like you can set different max packet sizes when the near end is the server versus the client.

Reply:
Yes, that's what I thought.  But I don't see how Quiche is allowing us to configure those values differently depending on whether the near end is the client or the server.  If we can't, we only need two new config variables, not four.

Reply:
Oh you mean different packet sizes for inbound connection and outbound connection? We'll need another quiche_config object for outbound connections. That is why we use `_in` settings here. We'll have similar code that use `_out` settings somewhere else in the future.